Index: consolekit-0.3.0/doc/dbus/Makefile.am
===================================================================
--- consolekit-0.3.0.orig/doc/dbus/Makefile.am	2009-03-03 01:56:14.000000000 +0100
+++ consolekit-0.3.0/doc/dbus/Makefile.am	2009-03-03 01:56:42.000000000 +0100
@@ -16,7 +16,7 @@
 htmldoc_DATA = ConsoleKit.html
 
 ConsoleKit.html : $(SPEC_XML_FILES)
-	$(XMLTO) xhtml-nochunks -m $(top_srcdir)/doc/dbus/config.xsl ConsoleKit.xml
+	$(XMLTO) --skip-validation xhtml-nochunks -m $(top_srcdir)/doc/dbus/config.xsl ConsoleKit.xml
 
 endif # DOCBOOK_DOCS_ENABLED
 
Index: consolekit-0.3.0/doc/dbus/Makefile.in
===================================================================
--- consolekit-0.3.0.orig/doc/dbus/Makefile.in	2009-03-03 01:56:12.000000000 +0100
+++ consolekit-0.3.0/doc/dbus/Makefile.in	2009-03-03 01:57:03.000000000 +0100
@@ -427,7 +427,7 @@
 
 
 @DOCBOOK_DOCS_ENABLED_TRUE@ConsoleKit.html : $(SPEC_XML_FILES)
-@DOCBOOK_DOCS_ENABLED_TRUE@	$(XMLTO) xhtml-nochunks -m $(top_srcdir)/doc/dbus/config.xsl ConsoleKit.xml
+@DOCBOOK_DOCS_ENABLED_TRUE@	$(XMLTO) --skip-validation xhtml-nochunks -m $(top_srcdir)/doc/dbus/config.xsl ConsoleKit.xml
 
 org.freedesktop.ConsoleKit.Manager.ref.xml : $(top_srcdir)/src/org.freedesktop.ConsoleKit.Manager.xml spec-to-docbook.xsl
 	$(XSLTPROC) $(top_srcdir)/doc/dbus/spec-to-docbook.xsl $< | tail -n +2 > $@
