diff -ur ConsoleKit-0.3.0/data/ConsoleKit.conf ConsoleKit-ted/data/ConsoleKit.conf
--- ConsoleKit-0.3.0/data/ConsoleKit.conf	2009-03-04 14:02:53.000000000 -0600
+++ ConsoleKit-ted/data/ConsoleKit.conf	2009-03-04 14:03:21.000000000 -0600
@@ -41,6 +41,9 @@
            send_member="GetSeats"/>
     <allow send_destination="org.freedesktop.ConsoleKit"
            send_interface="org.freedesktop.ConsoleKit.Manager"
+           send_member="GetSessions"/>
+    <allow send_destination="org.freedesktop.ConsoleKit"
+           send_interface="org.freedesktop.ConsoleKit.Manager"
            send_member="GetSessionForCookie"/>
     <allow send_destination="org.freedesktop.ConsoleKit"
            send_interface="org.freedesktop.ConsoleKit.Manager"
